source/deprecations/v1.x.html.md

```diff
@@ -996,16 +996,44 @@ totalCount: Ember.computed('values.[]', function() {
 Because of changes in the way Ember renders components in 1.13, setting properties within an overridden `didInsertElement` method
 will result in a deprecation warning.
 
-In many cases you can move your logic later in the component lifecycle by implementing the `didRender` hook, on of
+In many cases you can move your logic earlier in the component lifecycle by implementing the `didReceiveAttrs` hook, one of
 [the new hooks introduced in 1.13](http://emberjs.com/blog/2015/06/12/ember-1-13-0-released.html#toc_component-lifecycle-hooks).
 
 ```js
-  didRender() {
+  didReceiveAttrs() {
     this._super(...arguments);
     this.set('myValue', value);
   }
 ```
 
+It may even be possible to move the change to `init`, where the value can only be set once.
+
+```js
+  init() {
+    this._super(...arguments);
+    this.set('myValue', myValue);
+  }
+```
+
+Don't see a `set` in your `didInsertElement`? It may be in `didUpdate` or `didRender`. Due to a bug, these run during the same process.
```

Comment:
The bug here is that they run and report the `didInsertElement` as the method causing the error.  They will (and should) be run roughly at the same time as `didInsertElement`, but the error/deprecation/whatever had the wrong method name.